klippy/extras/ina381.py

```diff
@@ -0,0 +1,21 @@
+# Current monitor circuit
+#
+# Copyright (C) 2024  Elias Bakken <elias@iagent.no>
+#
+# This file may be distributed under the terms of the GNU GPLv3 license.
+class ina381:
+    def __init__(self, config):
+        printer = config.get_printer()
+        ppins = printer.lookup_object('pins')
+        ppins.register_chip('ina381', self)
+
+        # Reset over current alarm
+        reset_pin = config.get('reset_pin', 'ar100:PF4')
```

Comment:
It would not make sense to use a default that contains a user specified mcu name.